src/plugins/intel_cpu/src/utils/debug_capabilities.h

```diff
@@ -165,9 +165,15 @@ static inline std::ostream& _write_all_to_stream(std::ostream& os, const T& arg,
 
 #    define DEBUG_ENABLE_NAME debug_enable_##__LINE__
 
+#    ifdef __PRETTY_FUNCTION__
```

Comment:
What about something like this to cover MS as well:
```c++
#if defined(__clang__) || defined(__GNUC__)
    #define DEBUG_FUNCTION_NAME __PRETTY_FUNCTION__
#elif defined(_MSC_VER)
    #define DEBUG_FUNCTION_NAME __FUNCSIG__
#else
    #define DEBUG_FUNCTION_NAME __func__  // Fallback
#endif
```

Reply:
Applied. Also, replaced similar JIT emitters specific macro with the new one